src/Searcher.php

```diff
@@ -126,12 +126,12 @@ public function allowEmptySearchQuery(): self
      * @param string $orderByColumn
      * @return self
      */
-    public function add($query, $columns = null, string $orderByColumn = 'updated_at'): self
+    public function add($query, $columns = null, string $orderByColumn = null): self
     {
         $modelToSearchThrough = new ModelToSearchThrough(
             is_string($query) ? $query::query() : $query,
             Collection::wrap($columns),
-            $orderByColumn,
+            $orderByColumn ?? (String) constant("{$query}::UPDATED_AT"),
```

Comment:
`$query` can be a string as well (see `is_string()`).

Maybe we can assign a temporary variable:
`$queryBuilder = is_string($query) ? $query::query() : $query`

And then use the `getUpdatedAtColumn()` method:
`$orderByColumn ?: $queryBuilder->getUpdatedAtColumn()`

Reply:
Thanks! I have updated the code but the tests are failing.